src/App.vue

```diff
@@ -1,28 +1,37 @@
 <template>
-  <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
-  </div>
+<div>
+  <h1 v-if = "posts==[]"  > Loading... </h1>
+  <give-post v-else v-for="post in posts" :key="post.id" :title="post.title" :body="post.body">  </give-post>
+</div>
 </template>
 
-<script>
-import HelloWorld from './components/HelloWorld.vue'
 
+
+<script>
+/* eslint-disable */
+import axios from "axios"
+import givePost from "./components/givePost"
 export default {
   name: 'app',
-  components: {
-    HelloWorld
-  }
+  components:{
+    givePost
```

Comment:
@ChoudharyHamza make meaningful component names.

Reply:
ok will keeo in mind and have changed the name from givePost to RenderPosts

Reply:
also added the pagination feature.